docs/reference/triggers/v3iostream.md

```diff
@@ -100,56 +100,84 @@ As of Nuclio v1.1.33 / v1.3.20, you can configure the following configuration pa
+For test purposes, the ingested record data in the example is a string denoting the shard and record IDs - `shard-$shard_id-record-$record_id` (`--data`).
+As in the `create stream` command, the stream path is set to a "test-stream-0" stream in the root directory of the "users" data container, by using the stream-path argument and the `--container` option.
+```sh
 for shard_id in {0..31}
 do
     for record_id in {1..10}
     do
-        ./v3ctl create stream record \
+        ./v3ctl create stream record /test-stream-0 \
             --container users \
             --shard-id $shard_id \
-            --data shard-$shard_id-record-$record_id /test-stream-0
+            --data shard-$shard_id-record-$record_id
     done;
 done;
 ```
 
-The stream is ready for consumption by a Nuclio function. Deploy a Python function that logs the body and sleeps for 5 seconds:
+After the command completes successfully, the stream is ready for consumption by a Nuclio function. To test this, do the following from the dashboard's **Projects** page, to define and deploy a function that consumes the stream records and uses a `v3ioStream` trigger.
 
-```python
-import time
+> **Note:** Before you deploy the function, ensure that the log-forwarder service is enabled (see the **Services** dashboard page), so that you can view the function logs.
 
+1.  Select and existing project or create a new project, and then create a new Python function. Select to create the function from scratch.
 
-def handler(context, event):
-    context.logger.debug_with('Got event', shard_id=event.shard_id, body=event.body.decode('utf-8'))
-    time.sleep(5)
-```
+2.  On the function page, in the **Code** tab, set **Code entry type** to "Source code (edit online)" (default), and enter the following code in the **Source code** text box to define a function that logs the shard-ID event body and sleeps for 5 seconds:
+    ```python
+    import time
+
+
+    def handler(context, event):
+        context.logger.debug_with('Got event', shard_id=event.shard_id, body=event.body.decode('utf-8'))
+        time.sleep(5)
+    ```
+
+3.  Select the **Triggers** tab, and then select **Create a new trigger**.
+    In the **Name** text box, enter the trigger name "V3IO stream", and in the **Class** field select "V3IO stream" from the drop-down list.
+    Use the following trigger configuration:
 
-Under `Triggers`, create a `V3IO stream` with the following configuration:
-* URL: http://v3io-webapi:8081/users/test-stream-0@cg0
-* Max Workers: 8 (can be more, but here we'll have 8 workers reading all the shards)
-* Seek To: Earliest (if we use `Latest`, we'll never read the records we put into the stream - only records we create after deploy)
-* Password: Your access key
+    - **URL** - `http://v3io-webapi:8081/users/test-stream-0@cg0`.
+    - **Max Workers** - `8`.
```

Comment:
> but then you'll have 8 workers reading all the shards.
unsure what you mean here. my line was also pretty unclear, but i'm trying to say this number simply specifies how many workers handle all the shards.

Reply:
I didn't actually mean to alter the original version except for minor rephrasing,, but in retrospect, I see that I unintentionally did ... . => I've rephrased to the following — commit b5c3da6:
>**Max Workers** - `8`. This value signifies the number of workers assigned to handle all the shards. You can also set it to a different number.